### PR TITLE
Remove remaining guava usages and dependencies.

### PR DESCRIPTION
**Pull request by Chris-V**

### CODE REVIEW

File: gwtp-core/gwtp-dispatch-rest-shared/pom.xml [hunk 1]
```diff
@@ -53,7 +53,6 @@
         <dependency>
             <groupId>com.google.guava</groupId>
```
Comment (christiangoudreau) [inline gwtp-core/gwtp-dispatch-rest-shared/pom.xml]: You kept this one?

Comment (Chris-V) [inline gwtp-core/gwtp-dispatch-rest-shared/pom.xml]: test scoped in the root pom

**Comment by christiangoudreau** [comment]
LGTM

**Comment by meriouma** [comment]
LGTM

**Pull request merged**